frontend/src/v5/ui/routes/viewer/theme.ts

```diff
@@ -134,6 +134,18 @@ export const theme = createTheme(
 							'&.react-autosuggest__suggestions-container': {
 								boxShadow: 'none',
 							},
+							'&.color-picker__panel': { // Colour picker modal
```

Comment:
If you are writing the color picker style in the theme, could you please get rid of this file then ? `frontend\src\v5\ui\v4Adapter\dialogs\colorPickerDialog.overrides.ts`

It was used to tackle the color-picker style in the screenshot dialog, but if you put the style in a theme, it is no longer necessary

Reply:
Ah, I hadn't seen that file. Only the colorPicker.overrides file